### PR TITLE
(SIMP-4340) Update to simp/iptables 6.1.1

### PR DESCRIPTION
**Pull request by jeefberkey**
- Unset $iptables_docker_chain by default
  - The simp/iptables module has been updated so that this is not needed

### CODE REVIEW

**Comment by jeefberkey** [comment]
✔️ https://gitlab.com/simp/pupmod-simp-simp_docker/pipelines/17110435

**Comment by jeefberkey** [comment]
I plan on releasing this module after this PR is merged.

File: data/common.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@
 simp_docker::release_type: redhat
 simp_docker::manage_sysctl: true
 simp_docker::bridge_dev: docker0
+simp_docker::iptables_docker_chain: false
```
Comment (trevor-vaughan) [inline data/common.yaml]: This is not required any longer.

File: manifests/init.pp [hunk 2]
```diff
@@ -37,7 +38,7 @@
   Hash $default_options,
   Optional[Hash] $options,
 
-  Boolean $iptables_docker_chain = simplib::lookup('simp_options::firewall', { 'default_value' => false }),
+  Boolean $iptables_docker_chain,
```
Comment (trevor-vaughan) [inline manifests/init.pp]: This can be removed, along with the associated logic now.

Comment (jeefberkey) [reply manifests/init.pp]: I left it in in case users aren't able to get the newest iptables module, like if they're stuck on 6.0.0 or 6.1.0

Comment (trevor-vaughan) [reply manifests/init.pp]: You mean...if they do something that is invalid per the `metadata.json`?

They *need* that `iptables` update due to the bug fixes. Let's not release a new module that works around fixed bugs if we can help it.

File: manifests/init.pp [hunk 3]
```diff
@@ -24,8 +24,9 @@
 #
 #   This parameter will overwrite and default setting in $default_options.
 #
-# @param iptables_docker_chain If using the SIMP iptables module, add the
-#   `DOCKER` chain back when the iptables rules have been changed by Puppet.
+# @param iptables_docker_chain Add the `DOCKER` chain back when the iptables
```
Comment (lnemsick-simp) [inline manifests/init.pp]: This parameter does not exist

File: spec/acceptance/suites/ce/20_multi_node_spec.rb [hunk 4]
```diff
@@ -31,6 +31,8 @@ class { 'simp_docker':
     hosts.each do |host|
       it 'should apply with no errors' do
         apply_manifest_on(host, manifest, catch_failures: true, run_in_parallel: true)
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/ce/20_multi_node_spec.rb]: Is this expected behavior...to take 2 runs to be idempotent?

File: spec/acceptance/suites/default/10_one_node_spec.rb [hunk 5]
```diff
@@ -25,12 +25,12 @@
 
         on(host, 'yum install -y epel-release', run_in_parallel: true)
 
-        set_hieradata_on(host, { 'simp_options::firewall' => true })
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/default/10_one_node_spec.rb]: why did you remove this?

Comment (jeefberkey) [reply spec/acceptance/suites/default/10_one_node_spec.rb]: It used to be needed for the parameter I just removed

File: spec/acceptance/suites/ce/20_multi_node_spec.rb (hunk 4 above)
Comment (jeefberkey) [reply spec/acceptance/suites/ce/20_multi_node_spec.rb]: Yes, once to install and start docker, one to set the sysctl values which are only available after the docker daemon starts

**Comment by jeefberkey** [comment]
✔️ https://gitlab.com/simp/pupmod-simp-simp_docker/pipelines/17179136
